src/extension.ts

```diff
@@ -254,12 +254,18 @@ export async function activate(context: vscode.ExtensionContext) {
 
         ext.telemetry.assertPassiveTelemetry(ext.didReload())
     } catch (error) {
+        const stacktrace = (error as Error).stack?.split('\n')
+        // truncate if the stacktrace is unusually long
+        if (stacktrace !== undefined && stacktrace?.length > 40) {
```

Comment:
is `?` needed? the previous condition checks for `!== undefined`
```suggestion
        if (stacktrace !== undefined && stacktrace.length > 40) {
```

Reply:
You're right.  Removed and pushed.